test/lib/create-next-install.js

```diff
@@ -23,7 +24,9 @@ async function createNextInstall({
       const origRepoDir = path.join(__dirname, '../../')
       const installDir = path.join(
         tmpDir,
-        `next-install-${randomBytes(32).toString('hex')}${dirSuffix}`
+        `next-install${
+          name ? `-${name.toLowerCase().replace(/ /g, '-')}` : ''
```

Comment:
If the name includes certain chars it can break the tests so we should either normalize or revert this for now.